src/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1684,4 +1684,22 @@
   <data name="CryptoError_Sha1" xml:space="preserve">
     <value>Unexpected cryptographic error while computing SHA1 ({0}).</value>
   </data>
+  <data name="ClassLoad_General" xml:space="preserve">
+    <value>Could not load type '{0}' from assembly '{1}'.</value>
+  </data>
+  <data name="ClassLoad_MissingMethodRva" xml:space="preserve">
+    <value>Could not load type '{0}' from assembly '{1}' because the method '{2}' has no implementation (no RVA).</value>
```

Comment:
This error message is not quite right for CoreRT because of the type loading is more lazy ... the type can load just fine in CoreRT, despite this error. 

I do not think that the missing RVA deserves a dedicated message here. I would fold all failures during compilation of IL for given method that had to do with "bad IL" (including this one, or `CORJIT_BADCODE` returned by the JIT) into InvalidProgramException, with method name that had the problem. It would be order of magnitude improvement over CLR/CoreCLR - in CLR/CoreCLR, you typically do not get the method name in InvalidProgramException.